subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -327,6 +327,15 @@ Create a https://gradle.com/build-scans[build scan] with fine-grained informatio
 `-Dorg.gradle.debug=true`::
 Debug Gradle client (non-Daemon) process. Gradle will wait for you to attach a debugger at `localhost:5005` by default.
 
+`-Dorg.gradle.debug.port=(port number)`::
+Specifies the port number on which process is listening when debug is enabled. _Default is `5005`._
+
+`-Dorg.gradle.debug.server=true`::
+Then set to `true` and debugging enabled, Gradle will run the build with socked-attach mode of debugger. Otherwise socket-listen is used.
```

Comment:
```suggestion
If set to `true` and debugging is enabled, Gradle will run the build with the socket-attach mode of the debugger. Otherwise, the socket-listen mode is used.
```